dts/bindings/mmc/nxp,imx-usdhc.yaml

```diff
@@ -27,5 +27,12 @@ properties:
         property value should ensure the flags properly describe the signal
         that is presented to the driver.
 
+    no-1-8-v:
+      type: boolean
+      required: false
+      description:
+        When specified, denotes that 1.8V card voltage is not supported
```

Comment:
I think description should be phrased differently. This is taken from https://www.kernel.org/doc/Documentation/devicetree/bindings/mmc/mmc-controller.yaml (SPDX-License-Identifier: GPL-2.0)

Reply:
Thanks for review, modify it.